Added History section in Privacy whitepaper (#2760)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

The family group organizer may stop the data collection from the family safety portal.  For more information about Microsoft family safety features, see [What is a Microsoft family group?](https://support.microsoft.com/help/12413)

<!-- ====================================================================== -->
## Favorites

> Favorites now supports views that add an image thumbnail for favorites a user has saved.  All favorites data is stored locally on the device, and is organized per Microsoft Edge profile. Favorites are saved for the duration the favorite exists (10 years).  If you have sync turned on for favorites, your saved items are available across all signed-in and syncing versions of Microsoft Edge.

Author: ```suggestion
Favorites now supports views that add an image thumbnail for favorites that you've saved.  All favorites data is stored locally on the device, organized per Microsoft Edge profile. Favorites are saved for the duration the favorite exists (10 years).  If you have sync turned on for favorites, your saved items are available across all signed-in and syncing versions of Microsoft Edge.
```
changes i plan to make

> If you want Microsoft Edge not to capture images for your favorites, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Save Images for Favorites**.<!-- todo: no such control found in Canary 117 -->  All previously saved images will be erased.

Author: What's the label on the toggle?  I'd expect lowercase 'images'.  
Which Edge version will have the toggle?

In Canary 117, in `edge://settings/privacy` > **Services** section, **Save Images for Favorites** is not found.

My Canary 117 > Settings > Services (can't guarantee these are default toggle settings):
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/0c5d5d24-0375-46f3-aa50-ca575e9d6204)

Author: The new Favorites section was removed from this PR and moved to a dedicated PR 2771.  The rendered Favorites section in that PR
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2771#favorites
has latest planned label: "turn off the toggle for **Save images for Favorites**."

> If you want Microsoft Edge to not capture images for your favorites, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the button next to **Save Images for Favorites**.<!-- todo: no such control found in Canary 117 -->  All previously saved images will be erased.

Author: In Canary 117, in `edge://settings/privacy` > **Services** section, **Save Images for Favorites** is not found.

Author: Style guide (I might tweak UI text, after confirming the UI):
* "go to": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/navigate
* "toggle": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

Author: replaced by fresh comment https://github.com/MicrosoftDocs/edge-developer/pull/2760/files#r1283902474

**InPrivate** browsing uses the precise location permission setting of the profile from which the **InPrivate** session was launched. **Guest** mode always asks you for permission before granting the site your precise location.

<!-- ====================================================================== -->
## History

History now supports views to display images of your web navigations and enhance your history experience. All history data is stored locally on the device, and is organized per Microsoft Edge profile.  Images are saved for the duration the history exists (90 days).

> If you want Microsoft Edge to not capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Take Screenshots for History**.<!-- todo: in Dev 117 & Canary 117, there's instead a toggle labeled: **Save screenshots of site for History** -->  All previously saved images will be erased.

Author: In Dev 117 & Canary 117, in `edge://settings/privacy` > **Services** section, **Take Screenshots for History** is not found.  Instead there's a toggle labeled: **Save screenshots of site for History**.

Author: Style guide (I might tweak UI text, after confirming the UI):
* "go to": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/navigate
* "toggle": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

Author: replaced by fresh comment https://github.com/MicrosoftDocs/edge-developer/pull/2760/files#r1283903219

> If you want Microsoft Edge not to capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Take Screenshots for History**.<!-- todo: in Dev 117 & Canary 117, there's instead a toggle labeled: **Save screenshots of site for History** -->  All previously saved images will be erased.

Author: In Dev 117 & Canary 117, in `edge://settings/privacy` > **Services** section, **Take Screenshots for History** is not found.  Instead there's a toggle labeled: **Save screenshots of site for History**.

My Canary 117 > Settings > Services (can't guarantee these are default toggle settings):
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/0c5d5d24-0375-46f3-aa50-ca575e9d6204)

Reviewer: Hi Michael, thanks for that catch! Yes, the correct toggle name is "Save screenshots of site for History"
